final_task/rss_reader/other.py

```diff
@@ -0,0 +1,15 @@
+"""This module contain various help functions"""
+
+import sys
+
+
+def choice() -> list:
+    """Help the user to make choice"""
+
+    user_choice = input().split()
+    if user_choice[0] == 'q':
+        sys.exit()
```

Comment:
Same as below and ablove: it is better to have only one exit point from the application. Moreover our `rss_reader` is supposed to be 'one-shot' so if anything went wrong just stop the application.

Reply:
This realization gives a choice to the user: if he want to continue or leave.

Reply:
Again, the app is not supposed to work that way. Moreover it is better to avoid user inputs whenever we can because it is potential injection threat. Please remove this.

Reply:
The structure of the program was fixed, there is no more sys.exit(). It will be updated during the next commit.